.github/workflows/ci_test_cleanup.yml

```diff
@@ -8,17 +8,17 @@ jobs:
   Test-User-Data-Cleanup:
     environment: Testing
     runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        folder: [Corso_Restore_, TestRestore, '']
```

Comment:
Does this cover all calendars and contact patterns as well? I remember seeing some other event patterns last week along the lines to TestEvent, but maybe wrong.

Reply:
Here is some other patterns I see 

```
TestRestoreEvent:
TestRestoreEventObject:
```

Reply:
Should be covered by the prefix

Reply:
I don't see how. I think these were folders resulting from a restore test. There is no indication that they were wrapped in a Corso_Restore folder first

Output from Get-MailboxStatistics on the CI user
```
/Deleted Items                                                              19 403.8 KB (413,444 bytes)
/Deleted Items/Corso_Restore_28-Oct-2022_20:04:18                            2 76.42 KB (78,251 bytes)
/Deleted Items/Corso_Restore_28-Oct-2022_20-01-36.…                          2 76.45 KB (78,283 bytes)
/Deleted Items/Corso_Restore_28-Oct-2022_20-02-02.…                          3 40.71 KB (41,691 bytes)
/Deleted Items/Corso_Restore_28-Oct-2022_20-02-05.…                          5 67.85 KB (69,475 bytes)
/Deleted Items/TestRestoreEvent: 28-Oct-2022_20:22…                          1 39.75 KB (40,702 bytes)
/Deleted Items/TestRestoreEvent: 28-Oct-2022_20:36…                          1 39.77 KB (40,721 bytes)
/Deleted Items/TestRestoreEvent: 28-Oct-2022_20:43…                          1 10.46 KB (10,706 bytes)
/Deleted Items/TestRestoreEventObject: 28-Oct-2022…                          1 13.67 KB (13,993 bytes)
/Deleted Items/TestRestoreEventObject: 28-Oct-2022…                          1 13.67 KB (14,002 bytes)
/Deleted Items/TestRestoreEventObject: 28-Oct-2022…                          1 7.201 KB (7,374 bytes)
/Deleted Items/TestRestoreEventObject_28-Oct-2022_…                          1 17.82 KB (18,246 bytes)
```